README.md

```diff
@@ -34,7 +34,7 @@ Click on the table to download the most recent builds in the chosen language:
         </tr>
         <tr>
             <td>🇨🇿 Čeština</td>
-            <td align="center">40%</td>
+            <td align="center">60%</td>
```

Comment:
Double bonus!

Reply:
:D I felt like I need to bump it a bit since only Inferno campaign remains